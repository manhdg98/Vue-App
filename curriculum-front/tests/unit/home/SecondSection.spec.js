import { mount } from '@vue/test-utils'
import SecondSection from '@/components/home/SecondSection.vue'

describe('SecondSection.vue', () => {
  test('if snapshot is correct', () => {
    const wrapper = mount(SecondSection)
    expect(wrapper).toMatchSnapshot()
  })
})
