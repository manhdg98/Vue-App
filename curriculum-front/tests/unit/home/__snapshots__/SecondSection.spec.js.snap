// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SecondSection.vue if snapshot is correct 1`] = `
<div class="row home-section-two align-center">
  <div class="col-sm-4 offset-sm-2 col"><img src="img/user_dashboard.png" class="demo-screenshot"></div>
  <div class="col-sm-4 col">
    <h2>See your progress</h2>
    <p>
      Easily keep track of everything you are learning in one place.
      Instant progress feedback helps you stay on track.
    </p>
  </div>
</div>
`;
